src/language-js/print/flow.js

```diff
@@ -98,6 +99,10 @@ function printFlow(path, options, print) {
       return printIntersectionType(path, options, print);
     case "UnionTypeAnnotation":
       return printUnionType(path, options, print);
+    case "ConditionalType":
+      return printTernary(path, options, print);
+    case "InferType":
+      return ["infer", " ", print("typeParameter")];
```

Comment:
The naming inconsistency with other annotations isn't ideal. Flow will soon release a patch version that fixes this issue, and I will update the PR after that.